hanko-js/src/lib/HankoClient.ts

```diff
@@ -96,20 +102,32 @@ class HttpClient {
     return new Promise<Response>((resolve, reject) => {
       const controller = new AbortController();
       const timeout = setTimeout(() => controller.abort(), this.timeout);
+      const cookieName = "hanko";
+      const token = Cookies.get(cookieName);
 
       fetch(this.api + path, {
         mode: "cors",
         credentials: "include",
         headers: {
           Accept: "application/json",
           "Content-Type": "application/json",
+          Authorization: `Bearer ${token}`,
```

Comment:
This is not optimal, because if the cookie can not be read, the header looks like this `Authorization: Bearer undefined`. This feels a little bit buggy. We should not send this header, if the cookie is `undefined `.

Reply:
Fixed this one. I also rebased with main and did the same thing to the `_fetch2` function (Safari/iOS workaround).